internal/interpreter/dynamic_item_tree.rs

```diff
@@ -379,6 +379,10 @@ pub struct ItemTreeDescription<'id> {
     /// Map of all exported global singletons and their index in the compiled_globals vector. The key
     /// is the normalized name of the global.
     exported_globals_by_name: BTreeMap<String, usize>,
+
+    /// The type loader
+    #[cfg(feature = "highlight")]
+    pub(crate) type_loader: Option<std::rc::Rc<i_slint_compiler::typeloader::TypeLoader>>,
```

Comment:
Why is this an Option?
Ah ok it is there only for the root. Can this be documented?